docs/source/release-notes.rst

```diff
@@ -3,6 +3,210 @@ FiftyOne Release Notes
 
 .. default-role:: code
 
+FiftyOne Enterprise 2.11.0
+--------------------------
+*Released Aug 26, 2025*
+
+Includes all updates from :ref:`FiftyOne 1.8.0 <release-notes-v1.8.0>`, plus:
+
+- [Beta Release!] Introducing support for on-demand executors for Databricks
+  and Anyscale. Users can run any
+  :ref:`Delegated Operator <enterprise-delegated-operations>` on their
+  Databricks or Anyscale deployment by configuring an appropriate
+  :ref:`orchestrator <delegated-orchestrator-teamas>`.
+- Introduced support for
+  :ref:`distributed execution <enterprise-do-distributed-execution>` of a
+  :ref:`Delegated Operator <enterprise-delegated-operations>`. Execution of a
+  compatible Delegated Operator is split across multiple parallel workers,
+  enabling large operations to complete in less time. Initial support is for
+  operators without any pre- or post-processing steps (
+  :meth:`apply_model() <fiftyone.core.collections.SampleCollection.apply_model>`
+  and
+  :meth:`compute_metadata() <fiftyone.core.collections.SampleCollection.compute_metadata>`
+  ). Users can easily modify their own operators to support distribution by
+  following
+  :ref:`these instructions <writing-distributed-operators>`. More advanced
+  distribution support coming soon.
+- Disabled app server in-memory cache by default, as it is not compatible with
+  running multiple app server instances. Re-enable by setting
+  `fo.config.singleton_cache=True`.
```

Comment:
move this to open source section, it's disabled there also.
Additionally we may want to mention you can disable for non-app usage also, it's just default on.

Reply:
yes open source also